src/lib/event-transformer.ts

```diff
@@ -0,0 +1,53 @@
+import { EventDto } from "@/contracts/common"
```

Comment:
We should probably have a quick discussion on where we store models / dtos. Just so that we are consistent across the board :)